app/controllers/api/api_controller.rb

```diff
@@ -70,16 +75,31 @@ def server_info
       }
     end
 
-    def product_info
+    def product_info_data
       {
         :name                 => Vmdb::Appliance.PRODUCT_NAME,
         :name_full            => I18n.t("product.name_full"),
         :copyright            => I18n.t("product.copyright"),
         :support_website      => I18n.t("product.support_website"),
         :support_website_text => I18n.t("product.support_website_text"),
+        :branding_info        => branding_info
       }
     end
 
+    def branding_image_path(custom, default)
+      custom || ActionController::Base.helpers.image_path(default)
+    rescue Sprockets::FileNotFound # UI isn't loaded, we don't want images
+      nil
+    end
+
+    def branding_info
+      {
+        :brand      => branding_image_path(Settings.server.custom_brand, 'layout/brand.svg'),
```

Comment:
Why pass `Settings.server.custom_brand` into `branding_image_path` if the method doesn't do anything with it?

Reply:
I don't really understand this, why it would not do anything with it? The `branding_image_path` returns the path to the custom or default brand depending on what's available.

Reply:
Does the method need the `branding_` prefix?  It is not branding specific.  If we don't need to look up the path for the value from settings, I would expect something like `:brand      => Settings.server.custom_brand || image_path('layout/brand.svg')`